packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.styles.ts

```diff
@@ -40,6 +40,23 @@ const getIconColor = (messageBarType: MessageBarType | undefined, palette: IPale
   return palette.neutralSecondary;
 };
 
+// Returns the additional class name to add to the root based on the type of MessageBar.
```

Comment:
For comments not within a function, just use jsdoc style so you double-dip on the effort of writing the documentation (it shows up in intellisense/mouseover)